Src/FluentAssertions/ExceptionAssertionsExtensions.cs

```diff
@@ -4,12 +4,12 @@
 using FluentAssertions.Execution;
 using FluentAssertions.Specialized;
 
+#pragma warning disable AV1755 // "Name of async method ... should end with Async"; Async suffix is too noisy in fluent API
```

Comment:
I think it's better to just disable that rule globally.

Reply:
done